src/mon/AuthMonitor.cc

```diff
@@ -33,6 +33,9 @@
 #include "include/stringify.h"
 #include "include/assert.h"
 
+#include "mds/MDSAuthCaps.h"
```

Comment:
could take this chance to fix the Signed-off-by line of "Jing Li", i know it was copied literally from the original commit. but it does not look right.